### PR TITLE
Stub new modules

### PR DESCRIPTION
**Pull request by scribam**
- cellGamePs1Emu
- cellSysconfPs1emu
- sysBdMediaId (which included sys_get_bd_media_id)
- sysConsoleId (which included sys_get_console_id)
- sysPs2Disc

### CODE REVIEW

File: rpcs3/Emu/Cell/Modules/sysConsoleId.cpp [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#include "stdafx.h"
+#include "Emu/Cell/PPUModule.h"
+
+logs::channel sysConsoleId("sysConsoleId", logs::level::notice);
+
+s32 sys_get_console_id()
+{
+    UNIMPLEMENTED_FUNC(sysConsoleId);
```
Comment (OverlordCW) [inline rpcs3/Emu/Cell/Modules/sysConsoleId.cpp]: Extra space.

File: rpcs3/Emu/Cell/PPUModule.h [hunk 2]
```diff
@@ -201,7 +203,10 @@ class ppu_module_manager final
 	static const ppu_static_module sceNpUtil;
 	static const ppu_static_module sys_io;
 	static const ppu_static_module libnet;
+	static const ppu_static_module sysBdMediaId;
+	static const ppu_static_module sysConsoleId;
 	static const ppu_static_module sysPrxForUser;
+    static const ppu_static_module sysPs2Disc;
```
Comment (OverlordCW) [inline rpcs3/Emu/Cell/PPUModule.h]: extra space

File: rpcs3/Emu/Cell/Modules/sysConsoleId.cpp (hunk 1 above)
Comment (scribam) [inline rpcs3/Emu/Cell/Modules/sysConsoleId.cpp]: Thank you for the review.

File: rpcs3/Emu/Cell/PPUModule.h (hunk 2 above)
Comment (scribam) [inline rpcs3/Emu/Cell/PPUModule.h]: Yep, fixed, thanks.

**Comment by Nekotekina** [comment]
Stop spamming new cpp files. It everything fits into sysPrxForUser.cpp

**Comment by scribam** [comment]
So do you want me to copy the content of these cpp files inside sysPrxForUser.cpp ?
I thought it was 1 cpp file (+ sometimes an h file) for 1 module, is it different in fact ?

**Comment by Nekotekina** [comment]
There is no such rule. These small files are dead weight.

**Comment by scribam** [comment]
Understood.

**Pull request closed without merging**